domains/tronghieu.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "Nhth712"
+    },
+    "record": {
+        "CNAME": "ghs.googlehosted.com"
```

Comment:
404 Site. Please send over a preview. Reopen this PR when you have this resolved.